### ISSUE TITLE
Give note before changing the package namespace 

### ISSUE THREAD

**Issue by neuhalje** [vanitasvitae/pgpainless#1]
(Reminder)

If you plan to release the artefacts on maven etc you'll need to change the maven coordinates and the package namespace (to prevent collisions).

Please give me a quick note before you do that so I can scavenge all  changes interesting for bouncy-gpg :-)

**Comment by vanitasvitae** [comment]
Yeah, right now I'm a little hesitating to publish releases, as I have never done that before and I'm afraid to make mistakes :D so don't worry.

When the time is right, I'll surely ping you to take a look :)
